core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,123 @@
+// Default configuration for the org.virtuslab.ideprobe.config.IdeProbeConfig
+// Comments starting with the # sign are for non-default configs which can be used by users.
+// Comments starting with the // signs are for descriptions and guides.
+probe {
+
+  // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
+  intellij {
+    plugins = []
+    version {
+      // `release` is the "Version" and connected `build` is the "Build Number" of IntelliJ IDEA version. Both should
+      // be provided and they should match. You can search for proper string pairs here:
+      // https://www.jetbrains.com/intellij-repository/releases/ under "com.jetbrains.intellij.idea" heading.
+      // If you want to change the default intellij version, provide both parameters in your .conf file.
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    // Use below config with proper path if you want to use an existing Intellij instead of downloading a version.
+    // Example path from IntelliJ installed on macOs:
+    # path = "/Applications/IntelliJ\ IDEA\ CE.app/Contents"
+  }
+
+  // `workspace` is for `Option[org.virtuslab.ideprobe.config.WorkspaceConfig]` - default None,
+  // hence comments added as a reference. You can specify the workspace by using the below configurations.
+  # workspace {
+  // First option is to use a Default workspace - by just specifying `probe.workspace.path` with a valid string.
+  // `probe.workspace.path` can point to:
+  // a) a path on local filesystem (file or jar)
+  // or
+  // b) an online resource (http/https) - which can be for example a github repository as below:
```

Comment:
It can be also resources from classpath I think the syntax is `classpath:/my/resource/dir`